client/app/bundles/course/duplication/pages/Duplication/ItemsSelector/VideosSelector.jsx

```diff
@@ -57,24 +57,36 @@ class VideosSelector extends React.Component {
     );
   }
 
-  render() {
+  renderBody() {
     const { videos } = this.props;
-    if (!videos) { return null; }
+
+    if (videos.length < 1) {
```

Comment:
Can I suggest that we hide bulk selectors for videos.length <= 1, since it doesn't make sense to have the bulk selector for 1 video only?

Reply:
Added branching below to fix this. Added for achievements and surveys as well